libs/entry-components/styles/_generator.scss

```diff
@@ -2,6 +2,7 @@
 @use 'modules/components/forms/generator' as forms;
 @use 'modules/components/search-filter/generator' as search-filter;
 @use 'modules/components/tables/generator' as table;
```

Comment:
I assume that 'dialog' and 'table' modules are not in the plural form because they accurately represent singular entities within theming?

Reply:
They should be plural. Will be fixed.